src/resources/js/ian-client.js

```diff
@@ -79,10 +79,12 @@
 			switch (e.target.dataset.trigger) {
 				case "iconIan":
 					Ian.sidebar.classList.toggle("is-hidden");
+					Ian.icon.classList.toggle("hover");
```

Comment:
I'd be explicit here -you remove below but you toggle here. Why not `add`?
Same for `is-hidden`

Reply:
If nothing else, a double-click with add then remove the class.
`.add()` won't add duplicate classes.